Main_project_Helpers.rb

```diff
@@ -1,17 +1,16 @@
 require 'selenium-webdriver'
 require_relative 'User'
-
+require_relative 'Project'
+require_relative 'Issue'
 
 
  module Main_project_Helpers
 
 
 
 	def register_user(user_data)
```

Comment:
if you use this as an argument:



```
user_data=nil
```

you won't need to call this method like this:


```
register_user({})
```

just


```
register_user
```
and that's it

(this is a mistake)